adapterearthengine/routes/api/v1/earth_engine_router.py

```diff
@@ -17,11 +17,14 @@
 def build_query(rq):
     dataset = rq.get_json().get('dataset').get('data')
     sql = rq.args.get('sql', None) or rq.get_json().get('sql', None)
+    logging.debug(f'[ROUTER build_query]: {str(rq.args)}')
```

Comment:
perhaps use `info` instead here?

Reply:
Changed as requested on the next commit: https://github.com/resource-watch/adapter-earth-engine/pull/12/commits/ef6191631dfaf8c6c50d1356d83dd2bd129b3462